pm/controllers/storage.py

```diff
@@ -0,0 +1,96 @@
+""" PM controllers
+"""
+import os
+import re
+import shutil
+
+from cement.core import controller
+
+from pm.controllers import BaseController
+from pm.utils import filesystem, misc
+
+class StorageController(BaseController):
+    """ Storage Controller
+
+    Entry point for all functionalities related to storage
+    """
+    class Meta:
+        label = 'storage'
+        description = "Entry point for all functionalities related to storage"
+        stacked_on = 'base'
+        stacked_type = 'nested'
+        arguments = [
+            (['-r', '--run'], dict(type=str, help="Work with a specific run")),
+            (['-d', '--days'], dict(type=int, default=10, help="Days to consider a run \"old\""))
+        ]
+
+    #######################
+    # Storage subcommands #
+    #######################
+
+    @controller.expose(help="Move old runs to nosync directory so they're not synced to the processing server")
+    def cleanup(self):
+        for data_dir in self.app.config.get('storage', 'data_dirs'):
+            with filesystem.chdir(data_dir):
+                for run in [r for r in os.listdir(data_dir) if re.match(filesystem.RUN_RE, r)]:
```

Comment:
Is this only meant for high-seq runs ? as you defined `RUN_RE = '\d{6}_[a-zA-Z\d\-]+_\d{4}_[AB][A-Z\d]{9}'`. So it would not work for miseq, would it ?